tests/unit/io/Cereal_unittest.cc

```diff
@@ -0,0 +1,77 @@
+#include <shogun/lib/common.h>
+#include <shogun/base/Parameter.h>
+#include <gtest/gtest.h>
+#include <../tests/unit/base/CerealObject.h>
+
+#include <shogun/lib/SGVector.h>
+#include <cereal/archives/json.hpp>
+#include <shogun/io/SerializableJsonFile.h>
+#include <fstream>
+
+using namespace shogun;
+
+TEST(Cereal, Json_SGVector_FLOAT64_save_load_pair)
+{
+	SGVector<float64_t> a(2), b;
+	a.set_const(1.14263158);
+
+	{
+		std::ofstream os("cereatltest_sgvector.cereal");
+		cereal::JSONOutputArchive archive(os);
+		archive(a);
+	}
+
+	{
+		std::ifstream is("cereatltest_sgvector.cereal");
+		cereal::JSONInputArchive archive(is);
+		archive(b);
+	}
+
+	for (index_t i = 0; i < 2; i++)
+		EXPECT_NEAR(a[i], b[i], 1E-15);
+}
+
+TEST(Cereal, Json_Any_save_method)
+{
+	SGVector<float64_t> vec(5);
+	vec.range_fill(0);
+	Any a(vec);
+
+	{
+		std::ofstream os("cereatltest_save_anyobject.cereal");
+		cereal::JSONOutputArchive archive(os);
+		archive(a);
+	}
+}
```

Comment:
This test works and I got:
```
{
    "value0": {
        "value0": {
            "value0": 5,
            "value1": 0,
            "value2": 1,
            "value3": 2,
            "value4": 3,
            "value5": 4
        }
    }
}
```
in which value0 is the length of the vector
value1 - 5 are the entries of the sgvector